modules/options/base.go

```diff
@@ -4,131 +4,39 @@
 package options
 
 import (
-	"fmt"
-	"io/fs"
-	"os"
-	"path/filepath"
-
-	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/assetfs"
 	"code.gitea.io/gitea/modules/setting"
-	"code.gitea.io/gitea/modules/util"
 )
 
-var directories = make(directorySet)
+func CustomAssets() *assetfs.Layer {
```

Comment:
Looks like all functions named `CustomAssets` could become a package internal function

Reply:
Some could be internal, but some others could be exposed to help to list custom files, eg: list the custom labels.

So I think it's fine to leave them as public.